libraries/lib-wave-track/WaveClip.h

```diff
@@ -226,12 +231,10 @@ class WAVE_TRACK_API WaveClip final : public XMLTagHandler
     * available from the returned view is queried through
     * `AudioSegmentSampleView::GetSampleCount()`.
     *
-    * @param ii identifies the channel
     * @param start index of first clip sample from play start
     * @pre `ii < GetWidth()`
     */
-   AudioSegmentSampleView
-   GetSampleView(size_t ii, sampleCount start, size_t length) const;
+   ChannelSampleViews GetSampleView(sampleCount start, size_t length) const override;
```

Comment:
If removing `ii` from the argument list is the right decision (I'm not decided), then you should also delete the `@pre` mentioning it

Reply:
However, I'm not yet sure I agree that `ii` should be removed as an argument.